local-cli/runAndroid/runAndroid.js

```diff
@@ -150,9 +152,9 @@ function tryInstallAppOnDevice(args, device) {
   }
 }
 
-function tryLaunchAppOnDevice(device, packageName, adbPath, mainActivity) {
+function tryLaunchAppOnDevice(device, packageNameWithSuffix, packageName, adbPath, mainActivity) {
   try {
-    const adbArgs = ['-s', device, 'shell', 'am', 'start', '-n', packageName + '/.' + mainActivity];
+    const adbArgs = ['-s', device, 'shell', 'am', 'start', '-n', packageNameWithSuffix + '/' + packageName +  + mainActivity];
```

Comment:
Yup.  Thanks for catching that.  You can see that the other shell am start (with the hardcoded   '.MainActivity') gets it right.

Can you make this fix, or should I do it?